src/constants.ts

```diff
@@ -1,8 +1,37 @@
-export const REPO_OWNER = 'electron';
-export const REPO_NAME = 'electron';
+export const REPOS = {
+  electron: {
+    owner: 'electron',
+    repo: 'electron',
+  },
+  node: {
+    owner: 'nodejs',
+    repo: 'node',
+  },
+  libcc: {
+    owner: 'electron',
+    repo: 'libchromiumcontent',
+  },
+};
+
+export const rollTargets = {
```

Comment:
let's be consistent and either use either SHOUT_CASE or camelCase for all exported constants. I don't mind which, but it should be the consistent :)